README.md

```diff
@@ -56,7 +58,7 @@ REACT
 
 > Site is still in production and can't be deployed for now
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://symphonious-horse-ad1952.netlify.app/)
```

Comment:
- [ ] @team your live demo works initially and then gets broken, kindly ensure it works or remove it since its not a project requirement